aws/ecs.mk

```diff
@@ -40,9 +40,14 @@ CMD_ECS_SERVICE_DOCKER_PUSH = \
 	$(DOCKER) push $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(TAG) && \
 	$(DOCKER) push $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(TAG_LATEST)
 
+# Checks every 5 seconds the task status and goes on once task status is not RUNNING
+ECS_SERVICE_TASK_ID_STATUS = printf "%s" "Task $(ECS_SERVICE_TASK_ID) is running ."; until [ $$($(AWS) ecs describe-tasks --cluster $(ECS_CLUSTER_NAME) --tasks arn:aws:ecs:$(AWS_REGION):$(AWS_ACCOUNT):task/$(ECS_CLUSTER_NAME)/$(ECS_SERVICE_TASK_ID) | jq -r '.tasks[].lastStatus') != RUNNING ]; do printf "%s" "."; sleep 5; done 
```

Comment:
I would name it `CMD_ECS_SERVICE_TASK_STOP_WAIT`,  or something like that, as the macros doesn't seem to print out a string at the end, but instead it just waits for the task to be not `RUNNING`

Reply:
agree